README.md

```diff
@@ -308,7 +308,7 @@ There is also an ongoing experimental project to make OCaml multiprocessor and m
 
 - [Higher-Rank Polymorphism in OCaml](http://devmusings.legiasoft.com/blog/2008/05/23/higher-rank_polymorphism_in_ocaml)
 - [mikmatch](https://github.com/mjambon/mikmatch) – OCaml pattern-matching extended with regexps
-- [Inlined records in constructors](http://www.lexifi.com/blog/ocaml/inlined-records-constructors)
+- [Inlined records in constructors](http://www.lexifi.com/ocaml/inlined-records-constructors/)
```

Comment:
Just if possible, since https works, might as well update it to point to that while we're at it.

Reply:
Good point; done.